pandas/core/arrays/timedeltas.py

```diff
@@ -442,7 +443,7 @@ def __truediv__(self, other):
 
         if isinstance(other, (timedelta, np.timedelta64, Tick)):
             other = Timedelta(other)
-            if other is NaT:
+            if other is NaT:  # TODO: use NaTD
```

Comment:
once we actually do this (which is easy to implement, but I didn't because we don't have test coverage yet) then a bunch of this code can be templated/de-duplicated.